docs/sources/explore/logs-integration.md

```diff
@@ -16,81 +16,88 @@ weight: 15
 
 Explore is a powerful tool for logging and log analysis. It allows you to investigate logs from different data sources including:
 
-- [Loki]({{< relref "../datasources/loki" >}})
-- [Elasticsearch]({{< relref "../datasources/elasticsearch" >}})
-- [Cloudwatch]({{< relref "../datasources/aws-cloudwatch" >}})
-- [InfluxDB]({{< relref "../datasources/influxdb" >}})
-- [Azure Monitor]({{< relref "../datasources/azure-monitor" >}})
+- [Loki](/docs/grafana/<GRAFANA_VERSION>/datasources/loki/)
+- [Elasticsearch](/docs/grafana/<GRAFANA_VERSION>/datasources/elasticsearch/)
+- [Cloudwatch](/docs/grafana/<GRAFANA_VERSION>/datasources/aws-cloudwatch/)
+- [InfluxDB](/docs/grafana/<GRAFANA_VERSION>/datasources/influxdb/)
+- [Azure Monitor](/docs/grafana/<GRAFANA_VERSION>/datasources/azure-monitor/)
 - [ClickHouse](https://github.com/grafana/clickhouse-datasource)
 
 With Explore, you can efficiently monitor, troubleshoot, and respond to incidents by analyzing your logs and identifying the root causes. It also helps you to correlate logs with other telemetry signals such as metrics, traces or profiles, by viewing them side-by-side.
 
-The results of log queries are displayed as individual log lines and as a graph showing the logs volume for the selected time period.
+The results of log queries display as individual log lines and as a graph showing the logs volume for the selected time period.
 
 ## Logs volume
 
-When working with data sources that support a full range logs volume, Explore automatically displays a graph showing the log distribution for all the entered log queries. This feature is currently supported by Elasticsearch and Loki data sources.
+When working with data sources that support a full range logs volume, Explore automatically displays a graph showing the log distribution for all submitted log queries. **This feature is currently supported by the Elasticsearch and Loki data sources.**
```

Comment:
Don't use bold for emphasis: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold

```suggestion
When working with data sources that support a full range logs volume, Explore automatically displays a graph showing the log distribution for all submitted log queries. **This feature is currently supported by the Elasticsearch and Loki data sources.**
```